app/controllers/admin_shelters_controller.rb

```diff
@@ -4,4 +4,9 @@ def index
     @shelters = Shelter.find_by_sql('SELECT * FROM shelters ORDER BY name DESC')
     @pending_shelters = Shelter.pending
   end
+
+  def show
+    @shelter = Shelter.find(params[:id])
+    @applications = @shelter.associated_applications
```

Comment:
Again with the idea of having only one instance variable per action could you instead have this in the view such as `@shelter.associated_applications.each do |application|`.

Reply:
100% yes.